dogpile/cache/api.py

```diff
@@ -75,7 +79,17 @@ def __init__(self, arguments):  # pragma NO COVERAGE
          passed to :func:`.make_registry`.
 
         """
-        raise NotImplementedError()
+
+        serializer = arguments.get("serializer")
```

Comment:
**mike bayer** (zzzeek) wrote:

i'm going to move these up to CacheRegion where they will be similar to KeyMangler, taking the default from the cache backend itself.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/dogpile.cache/+/2283